### PR TITLE
2to3CuckooSandbox

### PR DESCRIPTION
**Pull request by JudahSchwartz**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CuckooSandbox/Scripts/CuckooDetonateFile/CuckooDetonateFile.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+res = demisto.executeCommand("cuckoo-create-task-from-file", demisto.args())
```
Comment (tkatzir) [inline Packs/CuckooSandbox/Scripts/CuckooDetonateFile/CuckooDetonateFile.py:4]: Put the "meat" of the script in a `main` method

**Review by tkatzir — CHANGES_REQUESTED** [review]
Looks good. Some minor comments

File: Packs/CuckooSandbox/ReleaseNotes/1_1_2.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+
+#### Scripts
+##### CuckooDisplayReport
+Updated the Docker image to: *demisto/python3:3.10.6.33415*.
+
+####Deprecated the following scripts
```
Comment (yaakovi) [inline Packs/CuckooSandbox/ReleaseNotes/1_1_2.md:6]: follow the release notes guidelines

**Review by yaakovi — CHANGES_REQUESTED** [review]

File: Packs/CuckooSandbox/ReleaseNotes/1_1_2.md (hunk 2 above)
Comment (JudahSchwartz) [reply Packs/CuckooSandbox/ReleaseNotes/1_1_2.md]: done

File: Packs/CuckooSandbox/Scripts/CuckooTaskStatus/CuckooTaskStatus.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+commonfields:
+  id: CuckooTaskStatus
+  version: -1
+name: CuckooTaskStatus
+script: ''
+type: python
+subtype: python3
+tags:
+- cuckoo
+comment: Deprecated. Use the 'cuckoo-view-task' command instead.
```
Comment (yaakovi) [inline Packs/CuckooSandbox/Scripts/CuckooTaskStatus/CuckooTaskStatus.yml:10]: relevant for all
```suggestion
comment: Deprecated. Use the 'cuckoo-view-task' command instead.
deprecated: true
```

File: Packs/CuckooSandbox/ReleaseNotes/1_1_2.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+Updated the Docker image to: *demisto/python3:3.10.6.33415*.
+
+#### CuckooGetScreenshot (Deprecated)
+
+Updated the Docker image to: *demisto/python3:3.10.6.33415*.
+
+#### CuckooTaskStatus (Deprecated)
+
+Updated the Docker image to: *demisto/python3:3.10.6.33415*.
```
Comment (yaakovi) [inline Packs/CuckooSandbox/ReleaseNotes/1_1_2.md:18]: relevant for all
```suggestion
#### CuckooTaskStatus
- Deprecated. Use the 'cuckoo-view-task' command instead.
- Updated the Docker image to: *demisto/python3:3.10.6.33415*.
```

**Review by yaakovi — APPROVED** [review]
Specify in the README the script is deprecated.

**Review by yaakovi — APPROVED** [review]

**Review by tkatzir — APPROVED** [review]

**Pull request merged**